### PR TITLE
Add marmo dependency

### PR DESCRIPTION
**Pull request by jpgonzalezra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
   "dependencies": {
     "ccxt": "^1.18.997",
     "commander": "^2.20.0",
-    "web3": "^1.2.0"
+    "marmojs": "^0.2.19",
+    "nodemon": "^1.19.1",
```
Comment (rotcivegaf) [inline package.json]: we dont use this dependency

**Review by rotcivegaf — CHANGES_REQUESTED** [review]
Good PR
Can you remove the nodemon dependency?
And its ready to merge

File: package.json (hunk 1 above)
Comment (jpgonzalezra) [reply package.json]: Is true, I now remove that dependency

**Comment by jpgonzalezra** [comment]
Done @rotcivegaf  =)

**Pull request merged**